spec/lib/mvi/messages/find_profile_message_spec.rb

```diff
@@ -120,14 +120,36 @@
     end
 
     context 'missing arguments' do
-      it 'throws an argument error' do
+      it 'throws an argument error for missing key' do
         expect do
           MVI::Messages::FindProfileMessage.new(
             given_names: %w[John William],
             last_name: 'Smith',
             birth_date: Time.new(1980, 1, 1).utc
           )
-        end.to raise_error(ArgumentError, 'wrong number of arguments')
+        end.to raise_error(ArgumentError, 'required arguments are invalid')
+      end
+
+      it 'throws an argument error for empty value' do
+        expect do
+          MVI::Messages::FindProfileMessage.new(
+            given_names: %w[John William],
+            last_name: '',
+            birth_date: Time.new(1980, 1, 1).utc,
+            ssn: FFaker::SSN.ssn
+          )
+        end.to raise_error(ArgumentError, 'required keys are missing')
```

Comment:
Can we update the language here to say 'required values are missing' instead of keys?

Reply:
I decided to have it echo out a different error message for missing a key and a different one for missing a value so that we can have more specific data to troubleshoot on. I am game either way though.

@joeniquette Thoughts?

Reply:
Good point for more insights into troubleshooting.  I'm fine with either.